rest/models/tickers.go

```diff
@@ -420,3 +423,206 @@ type TickerEvent struct {
 type TickerChangeEvent struct {
 	Ticker string `json:"ticker"`
 }
+
+// ListShortInterestParams contains parameters for the ListShortInterest method.
+type ListShortInterestParams struct {
+	// Path parameters
+	IdentifierType string `validate:"required" path:"identifierType"`
+	Identifier     string `validate:"required" path:"identifier"`
+
+	// Query parameters
+	DateEQ  *Nanos `query:"date"`
+	DateLT  *Nanos `query:"date.lt"`
+	DateLTE *Nanos `query:"date.lte"`
+	DateGT  *Nanos `query:"date.gt"`
+	DateGTE *Nanos `query:"date.gte"`
+
```

Comment:
we actually added an ipo_status parameter which can be used to filter by ipo_status